_inc/client/components/masthead/style.scss

```diff
@@ -1,25 +1,29 @@
 .jp-masthead {
 	background-color: $green-secondary;
 	text-align: center;
+
+	@media (max-width: rem( 720px ) ) {
```

Comment:
@beaulebens for readability  within our scss, anytime we referenace our REM function we have spaces between the parentheses and the px value. ie: `margin: rem( 8px ) 0;` This can be seen within just about every component stylesheet.